server/e2e/integration_asset_test.go

```diff
@@ -0,0 +1,71 @@
+package e2e
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/reearth/reearth-cms/server/internal/app"
+	"github.com/reearth/reearth-cms/server/pkg/id"
+)
+
+// GET|/assets/{assetId}
+func TestIntegrationGetAssetAPI(t *testing.T) {
+	e := StartServer(t, &app.Config{}, true, baseSeeder)
+
+	e.GET("/api/assets/{assetId}", id.NewAssetID()).
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.GET("/api/assets/{assetId}", id.NewAssetID()).
+		WithHeader("authorization", "Bearer secret_abc").
+		Expect().
+		Status(http.StatusUnauthorized)
+
+	e.GET("/api/assets/{assetId}", id.NewAssetID()).
+		WithHeader("authorization", "Bearer "+secret).
+		Expect().
+		Status(http.StatusNotFound)
+
+	e.GET("/api/assets/{assetId}", aid).
+		WithHeader("authorization", "Bearer "+secret).
+		Expect().
+		Status(http.StatusOK).
+		JSON().
+		Object().Keys().
+		Contains("id", "projectId", "name", "url", "contentType", "previewType", "totalSize", "file", "createdAt")
```

Comment:
Checking only for the presence or absence of a key does not allow you to immediately notice when a problem occurs with the contents. The easiest way is to compare the whole object with the Equal method, but that may be difficult. In that case, perform an assertion only on comparable objects as follows.

```go
	obj = e.GET("/api/models/{modelId}/items", ikey).
		WithHeader("authorization", "Bearer "+secret).
		WithQuery("page", 1).
		WithQuery("perPage", 5).
		Expect().
		Status(http.StatusOK).
		JSON().
		Object()

	obj.Value("page").Equal(1)
	obj.Value("perPage").Equal(5)
	obj.Value("totalCount").Equal(1)

	a = obj.Value("items").Array()
	a.Length().Equal(1)
	a.First().Object().Value("id").Equal(itmId.String())
	a.First().Object().Value("fields").Equal([]any{})
	a.First().Object().Value("parents").Equal([]any{})
	a.First().Object().Value("refs").Equal([]string{"latest"})
```